package.json

```diff
@@ -15,8 +15,8 @@
     "test": "node --experimental-vm-modules node_modules/jest/bin/jest.js",
     "test:integration": "node --experimental-vm-modules node_modules/jest/bin/jest.js tests",
     "test:components": "cd tests/components && vite",
-    "e2e": "playwright test",
-    "e2e:debug": "playwright test --debug",
+    "e2e": "yarn build && playwright test",
```

Comment:
@baruchiro just to understand this line why do we need to build it?

Reply:
You have to build the extension before running the `e2e`, right?

See my comment here: https://github.com/os-scar/overlay/pull/84#discussion_r1203534838

Reply:
got it now